exporter/jaeger/lib/opentelemetry/exporter/jaeger/collector_exporter.rb

```diff
@@ -37,8 +37,9 @@ def initialize(endpoint: ENV.fetch('OTEL_EXPORTER_JAEGER_ENDPOINT', 'http://loca
         # @param [Enumerable<OpenTelemetry::SDK::Trace::SpanData>] span_data the
         #   list of recorded {OpenTelemetry::SDK::Trace::SpanData} structs to be
         #   exported.
+        # @param [optional Numeric] timeout An optional timeout in seconds.
         # @return [Integer] the result of the export.
-        def export(span_data)
+        def export(span_data, timeout: nil)
```

Comment:
Note that we ignore the `timeout:` parameter here because we don't have a clean way to reach into the transport to adjust lower-level timeouts. I think that's a good "future work" item to figure out, but I don't think it should be in scope for this PR.